docs/constants/navigation.js

```diff
@@ -113,6 +113,17 @@ const general = [
     makePage('development/upgrading.mdx'),
     makePage('development/troubleshooting.mdx'),
   ]),
+  makeSection('Expo Modules API', [
+    makePage('modules/overview.mdx'),
+    makePage('modules/get-started.mdx'),
+    makePage('modules/native-module-tutorial.mdx'),
+    // makePage('modules/native-view-tutorial.mdx'),
```

Comment:
```suggestion
```

Let's not leave commented lines in.

Reply:
i plan to follow up on this today or tomorrow so i'm going to leave it there

Reply:
actually i'll just remove it :P thanks